src/kernel/src/arch/aarch64/context.rs

```diff
@@ -110,7 +110,7 @@ impl ArchContext {
     }
 
     pub fn change(&self, _cursor: MappingCursor, _settings: &MappingSettings) {
-        todo!("change")
+        // TODO: change page table entry
```

Comment:
Don't forget! :)

Reply:
Yes, I am going to dedicate a separate PR for TLB consistency stuff (unmap, change, etc.)